.drone/drone.jsonnet

```diff
@@ -353,7 +353,7 @@ local manifest(apps) = pipeline('manifest') {
         image: 'koalaman/shellcheck-alpine:stable',
         commands: ['apk add make bash && make lint-scripts'],
       },
-      make('loki', container=false) { depends_on: ['clone'] },
+      make('loki', container=false) { depends_on: ['clone', 'check-generated-files'] },
```

Comment:
this is necessary because previously we ran `check-generated-files` at the same time as `make loki`, the former runs a clean operation which removes the protos and then the `make loki` would fail because the protos were missing. This worked previously because the `make loki` step had a dependency to generate the protos and this PR removes that.